iroha-cpp/GrpcResponseHandler.cpp

```diff
@@ -0,0 +1,73 @@
+#include "GrpcResponseHandler.hpp"
+#include "logger/logger.hpp"
+#include "logger/logger_manager.hpp"
+
+
+using namespace grpc;
+
+
+namespace IROHA_CPP
+{
+
+
+GrpcResponseHandler::GrpcResponseHandler(logger::LoggerManagerTreePtr log_manager)
+    : tx_handler_(log_manager->getChild("Transaction")->getLogger()),
+      query_handler_(log_manager->getChild("Query")->getLogger()),
+      log_(log_manager->getChild("Grpc")->getLogger())
+{
+    handler_map_[CANCELLED] = "Operation canceled";
+    handler_map_[UNKNOWN] = "Unknown error";
+    handler_map_[INVALID_ARGUMENT] = "INVALID_ARGUMENT";
```

Comment:
Convention: first two are like sentence, but rest is UPPER_CASE

Reply:
Outdated comment.